src/grasp_data.cpp

```diff
@@ -111,18 +110,13 @@ bool GraspData::loadGraspData(const ros::NodeHandle& nh, const std::string& end_
   rosparam_shortcuts::get(parent_name, child_nh, "joints", joint_names);
   rosparam_shortcuts::get(parent_name, child_nh, "pregrasp_posture", pre_grasp_posture);
   rosparam_shortcuts::get(parent_name, child_nh, "grasp_posture", grasp_posture);
-  rosparam_shortcuts::get(parent_name, child_nh, "grasp_pose_to_eef_transform", grasp_pose_to_eef_transform);
+  rosparam_shortcuts::get(parent_name, child_nh, "grasp_pose_to_eef_transform", grasp_pose_to_eef_pose_);
   rosparam_shortcuts::get(parent_name, child_nh, "grasp_padding_on_approach", grasp_padding_on_approach_);
   rosparam_shortcuts::get(parent_name, child_nh, "max_finger_width", max_finger_width_);
   rosparam_shortcuts::get(parent_name, child_nh, "min_finger_width", min_finger_width_);
 
   // Convert generic grasp pose to this end effector's frame of reference, approach direction for short
 
-  // Orientation
-  ROS_ASSERT(grasp_pose_to_eef_transform.size() == 6);
```

Comment:
isn't grasp_pose_to_eef_pose_ still size 6? enforce this

Reply:
Already discussed offline

Reply:
Rosparam supports loading 6 values into an affine3d container and takes care of this check internally.